src/brainlayer/eval/labeling/enrichment-labeler.html

```
@@ -0,0 +1,318 @@
+  }
+</style>
+</head>
+<body>
+<header>
+  <h1>BrainLayer enrichment gold labeler</h1>
+  <div class="progress"><span id="count">0 / 0</span><div class="bar"><div id="bar"></div></div></div>
+  <label class="button" for="goldfile">Load enrichment-gold.jsonl</label>
+  <input id="goldfile" type="file" accept=".jsonl,.json" hidden>
+  <label class="button" for="resumefile">Resume labels</label>
+  <input id="resumefile" type="file" accept=".jsonl,.json" hidden>
+  <button class="primary" id="save">Save gold-enrichment-labels.jsonl</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)